### DISCUSSION TITLE
Confirmation of unstarring if in list

### DISCUSSION THREAD

**Discussion by MattIPv4** [community/community#8238] [Other Feature Feedback, Questions, & Ideas]
I think a nice enhancement might be to have the user confirm that they want to unstar the repo if they also have it in a list. It isn't immediately apparent (at least, to me) that pressing the main star button to unstar a repo will also remove it from any lists you have it in (part of my mind was treating "main" starring as a default list, alongside other lists, but I also realise that makes no sense now).

--- 

Edit: It does look like this behaviour does exist on the mobile app (❤️ ), would be great to see it also happen on web :)

**Comment by bailey** [comment]
Totally agree, Matt! Thanks for dropping this feedback. I'll update here soon as we roll out more updates 👀
